uiowa/uiowa
# Classrooms - Accessibility Features

Resolves https://github.com/uiowa/uiowa/issues/6488

Holding off on adding the filter till a later slice per https://github.com/uiowa/uiowa/issues/6488#issuecomment-1599304704

# To Test

```
ddev blt ds --site=classrooms.uiowa.edu && ddev drush @classrooms.local uli admin/structure/taxonomy/manage/accessibility_features/add
```

There should be no sync/import errors.

Add some of these api mappings:

Podium - Height Adjustable
Table - ADA Student Desk
Assistive Device - Hearing Assist
Assistive Device - Loop
Table - 24" x 60" Adjustable Height
Power Door Opener

`ddev drush @classrooms.local classrooms-rooms`

Go back into the recently created terms and see the that there are rooms now associated with them. Click into one of the rooms and see the Accessibility Features field is placed as it currently is on PROD. Edit the room and see that the field itself is in the disabled imported section of the form.

<img width="576" alt="Screenshot 2023-06-23 at 2 38 05 PM" src="https://github.com/uiowa/uiowa/assets/4663676/f48b1f75-5506-4b05-bfe6-3a62a4c847af">

<img width="706" alt="Screenshot 2023-06-23 at 2 43 47 PM" src="https://github.com/uiowa/uiowa/assets/4663676/96197f9e-2f41-4e76-b37e-afbc60cd5864">
